voc/storage_service.go

```diff
@@ -27,7 +27,7 @@ package voc
 
 type StorageService struct {
 	*NetworkService
-	Storages	[]ResourceID `json:"storages"`
-	HttpEndpoint	*HttpEndpoint `json:"httpEndpoint"`
+	Storages *Storage `json:"storages"`
```

Comment:
We need an array of resource IDs. We cannot directly add a referenced resource object here because we might get cyclic dependencies when serialising them to JSON. That is also why we do some extra tricks in rego in #486. In the future, we might circumvent this with a graph database or something similar, but for now we need to refer to other resources purely by their ID.